fr_CA/home/src/main/resources/rdf/i18n/fr_CA/display/firsttime/aboutQrCodes.n3

```diff
@@ -0,0 +1,38 @@
+# $This file is distributed under the terms of the license in LICENSE$
+
+@prefix about: <http://vitro.mannlib.cornell.edu/ns/default/about#> .
+
+about:about_qr_codes_dg
+    a   <java:edu.cornell.mannlib.vitro.webapp.utils.dataGetter.FixedHTMLDataGetter> ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#htmlValue>
+        """
+        <h2>About QR Codes</h2>
+
+		<h3>What is QR?</h3>
+
+		<p>QR, which stands for <i>Quick Response</i>, is a standard for two-dimensional barcodes that can be read by mobile devices equipped with cameras (such as a smartphone).</p>
+
+		<h3>What kind of data is in a QR code?</h3>
+
+		<p>QR codes can encode many different types of data.  The QR code on the profile page of a person in VIVO encodes information about that person in <i>vCard</i> format.  This is a sort of digital business card, containing information about the person's name, address, telephone number, email address, and more.</p>
+
+		<p>If you click on a person's QR code in VIVO, you will be taken to the <i>QR Export</i> page for that person.  This page contains the vCard QR code seen on the front page, as well as a QR code containing a simple hyperlink to the user's VIVO profile.  It also contains an HTML tag for each code, that can be embedded in a web page to display that code on other sites.</p>
+
+		<h3>How do I read a QR code?</h3>
+
+		<p>There are apps you can download for the iPhone and Android phones that allow you to use your phone's camera to interact with QR codes placed on objects in the real world.</p>
+
+		<h3>What can I do with a QR code?</h3>
+
+		<p>Anything!  For a state-of-the-art business card, put your hyperlink QR code on the back of it.  That way anyone with a properly-equipped smartphone can quickly go to your VIVO profile by simply pointing their camera at the code.  Or put your vCard QR code on your office door to provide more contact information.</p>
+        """@fr-CA ;
+        <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#saveToVar>
+                "aboutQrCodesPage" .
+about:about_qr_codes
+    a   <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#Page> ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#hasDataGetter>
+        about:about_qr_codes_dg ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#title>
+        "About QR Codes"@fr-CA ;
+    <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#urlMapping>
+        "/about_qrcode" .
```

Comment:
not translated in French. There is translation in the deleted ftl file which should be copied here.